apps/greenhouse/package.json

```diff
@@ -45,12 +51,6 @@
     "clean:cache": "rm -rf .turbo"
   },
   "dependencies": {
-    "@cloudoperators/juno-app-doop": "*",
```

Comment:
That ones can be removed because we have the manifest?

Reply:
Please, let's focus on the fixing the changeset config in this PR, not improving